### PR TITLE
Add caps lock led support to DZ60

### PR DESCRIPTION
**Pull request by mechmerlin**
I can't believe this hasn't been reported until now! lol!

### CODE REVIEW

**Comment by Pieparker** [comment]
Cheers for requesting. Also, for posterity, much love #5253.

File: keyboards/dz60/dz60.c [hunk 1]
```diff
@@ -26,3 +26,13 @@ void led_set_kb(uint8_t usb_led) {
   // Set this per keymap, instead of globally
   led_set_user(usb_led);
 }
+
+void led_set_user(uint8_t usb_led) {
```
Comment (drashna) [inline keyboards/dz60/dz60.c]: This block really should be in `led_set_kb`

Comment (mechmerlin) [reply keyboards/dz60/dz60.c]: got it!

**Comment by drashna** [comment]
Could you make sure that `led_set_user` is called at the end of `led_set_kb`?

**Comment by mechmerlin** [comment]
Your wish is my command. =P

**Comment by drashna** [comment]
Thanks. :D

**Pull request merged**